test/Test/Universum/StringProps.hs

```diff
@@ -1,3 +1,8 @@
+-- Some tests in this module have a caveat: they depend on compiler
+-- optimizations and will likely fail if compiled with -O0.
+-- That's because they depend on rewrite rules (and tests these rules),
```

Comment:
```suggestion
-- That's because they depend on rewrite rules (and test these rules),
```

(I guess)

Reply:
Right, done, going to merge and make a release once CI passes.